TezosKit/Operation/TransactionOperation.swift

```diff
@@ -33,7 +33,11 @@ public class TransactionOperation: AbstractOperation {
    * @param parameters Optional parameters to include in the transaction if the call is being made to a smart contract.
    * @param operationFees OperationFees for the transaction. If nil, default fees are used.
    */
-  public convenience init(amount: TezosBalance, source: Wallet, destination: String, parameters _: [String: Any]? = nil, operationFees: OperationFees? = nil) {
+  public convenience init(amount: TezosBalance,
```

Comment:
```
public convenience init(
  amount: TezosBalance,
  source: Wallet,
  destination: String,
  parameters _: [String: Any]? = nil,
  operationFees: OperationFees? = nil
) 
```

Is the proper style

Reply:
Done.  The dangling `)` is super odd to me. 

Should I also be invoking methods using this same methodology (break after `(`, +2 then align params, then line break, then `)` ?)

Reply:
Likewise, I'm surprised the linter doesn't support catching this. Have you found an opt-in rule?

Reply:
Its Google style, but I think it allows for the ) to be on the previous line.  The args definitely go on the line after the ( though

Reply:
I pulled in a few rules which seem to help enforce this and fixed elsewhere. The dangling `(` seems to be what is supported by swiftlint and I value automatic consistency.